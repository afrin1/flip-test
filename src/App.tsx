import React from 'react'
import { NavigationContainer } from '@react-navigation/native';
import { createNativeStackNavigator } from '@react-navigation/native-stack';
import Transactions from './transactions/Transactions';
import TransactionDetail from './transactions/TransactionDetail';

const Stack = createNativeStackNavigator();

const App = () => {
  return (
    <NavigationContainer>
      <Stack.Navigator screenOptions={{ headerShown: false }}>
        <Stack.Screen
          name="TransactionList"
          component={Transactions}
        />
        <Stack.Screen name="TransactionDetail" component={TransactionDetail} />
      </Stack.Navigator>
    </NavigationContainer>
  );
}

export default App
