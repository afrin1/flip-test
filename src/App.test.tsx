import React from 'react'
import App from '../src/App'
import { render } from '@testing-library/react-native'

describe('App', () => {
  it('should match snapshot', () => {
    expect(render(<App />)).toMatchSnapshot()
  })
})
