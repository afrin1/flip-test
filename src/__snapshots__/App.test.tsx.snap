// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#C5C5C5",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "#FFFFFF",
          "borderRadius": 5,
          "flexDirection": "row",
          "marginLeft": 10,
          "marginRight": 10,
          "marginTop": 10,
        }
      }
    >
      <TextInput
        inlineImageLeft="search"
        onChangeText={[Function]}
        placeholder="Cari nama, bank, atau nominal"
        placeholderTextColor="#C5C5C5"
        style={
          Object {
            "color": "black",
            "flex": 1,
            "fontSize": 18,
          }
        }
        value=""
      />
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        nativeID="animatedComponent"
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "justifyContent": "center",
            "opacity": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FF6347",
              "fontSize": 14,
              "fontWeight": "bold",
              "margin": 10,
              "textAlignVertical": "center",
            }
          }
        >
          URUTKAN V
        </Text>
      </View>
    </View>
    <RCTScrollView
      contentContainerStyle={
        Object {
          "paddingBottom": 20,
        }
      }
      data={Array []}
      getItem={[Function]}
      getItemCount={[Function]}
      keyExtractor={[Function]}
      onContentSizeChange={[Function]}
      onLayout={[Function]}
      onMomentumScrollBegin={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      stickyHeaderIndices={Array []}
      style={
        Object {
          "marginLeft": 10,
          "marginRight": 10,
        }
      }
      viewabilityConfigCallbackPairs={Array []}
    >
      <View />
    </RCTScrollView>
  </View>
</RCTSafeAreaView>
`;
